src/service_daemon.rs

```diff
@@ -1578,18 +1547,9 @@ impl Zeroconf {
     /// Returns the list of interface IPs that sent out the announcement.
     fn send_unsolicited_response(&mut self, info: &mut ServiceInfo) -> Vec<IpAddr> {
         let mut outgoing_addrs = Vec::new();
-        // Send the announcement on one interface per ip version.
-        let mut multicast_sent_trackers = HashSet::new();
-
         let mut outgoing_intfs = Vec::new();
 
         for intf in self.my_intfs.iter() {
```

Comment:
Although I would love to see any removal of unneeded code, I don't find the original issue (in #239) was gone. 

It's true we no longer have separate `sock` for each intf, and use the common `ipv4_sock` and `ipv6_sock`.  But we are still looping through `self.my_intfs` here, and if an interface has multiple addresses, multiple `intf` will show up and we will send out packet for each of them. 

Maybe I've missed something, please let me know.

Reply:
As far as I understand the following loop: https://github.com/keepsimple1/mdns-sd/blob/8cb1a59387d5a3baaa6087d09b386be8783824eb/src/service_daemon.rs#L969, we only add one IPv4 and one IPv6 address per interface, as a second bind to the respective address family fails and is skipped. That's what I've observed under Linux, not sure about the behavior of other platforms.

Reply:
You are correct 👍 . I didn't realize that case, which would render the concerned code unneeded.